src/app/components/routing/application-routing.component.ts

```diff
@@ -22,6 +22,9 @@ export class ApplicationRoutingComponent implements OnInit{
     if(userRole.indexOf('jps-recorder') != -1){
```

Comment:
As part of this code we will need to change the manage sitting records screen so that the previous button performs the correct action

Reply:
Done